src/optsummary.jl

```diff
@@ -90,6 +90,34 @@ function OptSummary(
     )
 end
 
+"""
+    columntable(s::OptSummary, [stack::Bool=false])
+
+Return `s.fitlog` as a `Tables.columntable`.
+
+When `stack` is false (the default), there will be 3 columns in the result:
+- `iter`: the iteration number
```

Comment:
It's technically the sample number, i.e. `iteration / thin`

Reply:
Changed